modules/core/src/main/scala/pillars/modules.scala

```diff
@@ -50,7 +50,7 @@ end Loader
 object Loader:
     final case class Context[F[_]: Async: Network: Tracer: Console](
         observability: Observability[F],
-        configReader: Reader[F],
+        reader: Reader[F],
```

Comment:
I think `reader` is maybe too generic as it's only used for configuration. But I don't have any good idea...